app/javascript/packages/document-capture/components/acuant-capture.tsx

```diff
@@ -16,20 +16,16 @@ import type { ReactNode, MouseEvent, Ref } from 'react';
 import AnalyticsContext from '../context/analytics';
 import AcuantContext from '../context/acuant';
 import FailedCaptureAttemptsContext from '../context/failed-capture-attempts';
-import AcuantCamera from './acuant-camera';
+import AcuantCamera, { AcuantDocumentType } from './acuant-camera';
 import AcuantCaptureCanvas from './acuant-capture-canvas';
 import FileInput from './file-input';
 import DeviceContext from '../context/device';
 import UploadContext from '../context/upload';
 import useCounter from '../hooks/use-counter';
 import useCookie from '../hooks/use-cookie';
-import type {
-  AcuantSuccessResponse,
-  AcuantDocumentType,
-  AcuantCaptureFailureError,
-} from './acuant-camera';
+import type { AcuantSuccessResponse, AcuantCaptureFailureError } from './acuant-camera';
 
-type AcuantDocumentTypeLabel = 'id' | 'passport' | 'none';
+type AcuantDocumentTypeLabel = keyof typeof AcuantDocumentType;
```

Comment:
Actually I just realized now that this is lowercased, this type signature is a lie. I think I will remove it since we seem to be using it only for logging and not for app logic. For app logic we can continue passing around the enum

Reply:
removed in 082292392980e3135ad4bad29515b515a01070da